cmd/setup/35_2.sql

```diff
@@ -0,0 +1,3 @@
+DROP INDEX IF EXISTS eventstore.es_wm;
+
+ALTER INDEX eventstore.es_wm_temp RENAME TO es_wm;
```

Comment:
my local cockroach fails to rename in the same transaction: `error="ERROR: index name \"es_wm\" already exists (SQLSTATE 42P07)"`

Reply:
Solved now, each step runs in a seperate exec. I also cleaned up how we load multi-statement migrations and made that a bit more reusable.